pep-0484.txt

```diff
@@ -610,6 +611,21 @@ the runtime class of the objects created by instantiating them doesn't
 record the distinction. This behavior is called "type erasure"; it is
 common practice in languages with generics (e.g. Java, TypeScript).
 
+Class attribute of a generic class can be looked up only through the instance
```

Comment:
Hmm, I am not sure.

Somehow I would interpret `instance attribute` as referring to the physical object stored on an instance, and of course if there is one, we can't actually refer to the class attribute at all.